src/index.js

```diff
@@ -52,6 +52,7 @@ const Search = React.createClass({
     inputProps.placeholder = inputProps.placeholder || 'Search'
     return (
       <div className={className}>
+        <img width="15px" height="15px" src="svg/magnifying_glass_icon.svg" />
```

Comment:
you cannot do that on a npm module. We either need to inline it or not do it at all. I'm more leaning toward not doing at all so the user is free to do whatever they want. But we can change the example though